packages/core/src/fhir-deduplication/deduplicate-fhir.ts

```diff
@@ -266,6 +271,7 @@ const resourceFiltersMap = new Map<string, Map<string, ResourceFilter>>([
   ["Composition", compositionFiltersMap],
   ["Observation", observationFiltersMap],
   ["Procedure", procedureFiltersMap],
+  ["Immunization", immunizationFiltersMap],
```

Comment:
we werent removing dangling resources from Immunizations

Reply:
Good catch!

It would be good to double check we're doing it for all resources we're deduping - get a list either from the code or [this QA one](https://www.notion.so/metriport/List-of-resources-for-Dedup-QA-4c8d0c075e8c48989a90e2a6db58652c?pvs=4#4d8ed252bf094eefb4f7843059d51d7e).

Reply:
double check